src/k8s-extension/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.4.0
```

Comment:
yes, this will only have history and version bump change as we created this branch off k8s-extension/public and merged with upstream repo changes to get in sync and now merging with k8s-extension/public branch again